apps/extension/src/core/media/openrouter.ts

```diff
@@ -0,0 +1,45 @@
+import type { MediaModelConfig, RequestOptions } from "./model"
+import { MediaModel } from "./model"
+
+
+export function init(
+  config: Pick<MediaModelConfig, "debug" | "identifier"> &
+    Partial<Pick<MediaModelConfig, "cacheGet" | "cacheSet">>,
+  opts: RequestOptions
+): MediaModel {
+  return new MediaModel(
+    {
+      ...config,
+      defaultBaseUrl: "http://localhost:3003",
```

Comment:
this should be `${getExternalConfigURL()}/api/v1`

Reply:
changed!